app/javascript/components/modals/complete-exercise-modal/TutorialCompletedModal.tsx

```diff
@@ -40,17 +43,25 @@ export const TutorialCompletedModal = ({
       </div>
       <div className="btns">
         {/* TODO: Change to track.course */}
-        {track.numConcepts > 0 ? (
-          <a href={track.links.concepts} className="btn-primary btn-m">
+        {completion.track.numConcepts > 0 ? (
+          <a
+            href={completion.track.links.concepts}
+            className="btn-primary btn-m"
+          >
             <span>Show me the Concepts</span>
             <GraphicalIcon icon="arrow-right" />
           </a>
         ) : (
-          <a href={track.links.exercises} className="btn-primary btn-m">
+          <a
+            href={completion.track.links.exercises}
+            className="btn-primary btn-m"
+          >
             Show me more exercises
           </a>
         )}
-        <button className="btn">Return to “Hello, World!”</button>
```

Comment:
@iHiD I decided to remove the hardcoded exercise names in this modal. I remember that there was a TODO for this in another component. We might want to apply the same here.

Reply:
OK, cool. You need to put `{completion.exercise.title}` back in quotes in the button to match what this originally was.